e2e/tests/controlled-tier/ct-workspace-share.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { createWorkspace, findWorkspaceCard, openTab, signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import { makeRandomName } from 'utils/str-utils';
+import WorkspaceEditPage, { AccessTierDisplayNames } from 'app/page/workspace-edit-page';
+import WorkspaceAboutPage from 'app/page/workspace-about-page';
+import { LinkText, MenuOption, Tabs, WorkspaceAccessLevel } from 'app/text-labels';
+import ShareModal from 'app/modal/share-modal';
+import { waitWhileLoading } from 'utils/waits-utils';
+import WorkspacesPage from 'app/page/workspaces-page';
+import HomePage from 'app/page/home-page';
+import UnavailableTierModal from 'app/modal/unavailable-tier-modal';
+import DataAccessRequirementsPage from 'app/page/data-access-requirements-page';
+
+describe('Share Controlled Tier Workspace', () => {
+  const hasCTAccessUser = config.WRITER_USER;
+  const notCTUser = config.EGRESS_TEST_USER; // Without CT access
+
+  const workspaceName = makeRandomName('e2eShareControlledTierWorkspace');
+
+  test('User without CT access cannot create CT workspace', async () => {
```

Comment:
This is a nice test, thanks!  I would also like to see the flow for cancel modal - observe that the Create Workspace page forces the tier back to registered.

Don't need a new test for this - we can add this as an earlier step here.